.github/workflows/create-azure-self-hosted-runners.yml

```diff
@@ -0,0 +1,119 @@
+name: create-azure-self-hosted-runners
+
+on:
+  workflow_dispatch:
+    inputs:
+      amount_of_runners:
+        description: 'Amount of runners to set up'
+        required: true
+      runner_scope:
+        type: choice
+        required: true
+        description: Scope of the runner. On personal accounts, only "repo-level" works
+        options:
+        - org-level
+        - repo-level
+      runner_org:
+        type: string
+        required: true
+        description: Organization or personal account to deploy the runner to
+      runner_repo:
+        type: string
+        required: false
+        description: Repo to deploy the runner to. Only needed if runner_scope is set to "repo-level"
+
+env:
+  ACTIONS_RUNNER_ORG: "${{ github.event.inputs.runner_org }}"
+  ACTIONS_RUNNER_REPO: "${{ github.event.inputs.runner_repo }}"
+  # This has to be a public URL that the VM can access after creation
+  POST_DEPLOYMENT_SCRIPT_URL: https://raw.githubusercontent.com/git-for-windows/git-for-windows-automation/main/azure-self-hosted-runners/post-deployment-script.ps1
```

Comment:
This URL will only work after this PR has been merged.

Reply:
Obviously. We could use `${{ github.repository }}` and `${{ github.ref_name }}` to make it less specific, though.